### PR TITLE
sql/mssql: inspect checks

### PR DESCRIPTION
**Pull request by giautm**

### CODE REVIEW

File: sql/mssql/inspect.go [hunk 1]
```diff
@@ -110,6 +107,62 @@ type queryScope struct {
...
+			if _, ok := t.Column(column.String); !ok {
+				return fmt.Errorf("mssql: column %q was not found for check %q", column.String, name)
+			}
+		}
+		check, ok := names[name]
+		if !ok {
+			check = &schema.Check{
+				Name:  name,
+				Expr:  clause,
+				Attrs: nil,
```
Comment (a8m) [inline sql/mssql/inspect.go:149]:
```suggestion
```

File: sql/mssql/inspect.go [hunk 2]
```diff
@@ -110,6 +107,62 @@ type queryScope struct {
...
+			}
+			if column.Valid {
+				check.Attrs = append(check.Attrs, &CheckColumns{
+					Columns: []string{column.String},
+				})
+			}
+			if disabled == 1 {
```
Comment (a8m) [inline sql/mssql/inspect.go:156]: Can’t you use sql.NullBool instead, or just bool?

**Review by a8m — APPROVED** [review]
Minor comments. Well done 🚀

File: sql/mssql/inspect.go (hunk 2 above)
Comment (giautm) [reply sql/mssql/inspect.go:156]: I will update in next PR for other places too.

**Pull request merged**